### ISSUE TITLE
Polling for poll-only tokens fails on server

### ISSUE THREAD

**Issue by timosturm** [privacyidea/privacyidea#2744]
### Top-level intent

For a poll-only token, I tried to poll for new challenges using the authenticator app. (On dev server).

### Steps to reproduce

1. Set all policies for poll-only.
2. Enroll a 'push-token' on the server.
3. Poll for challenges using the app.

### Expected outcome

A valid answer is sent to the app, either with existing challenges or with no challenges.

### Actual outcome

The following response to the request is received by the app:
~~~~
Status code: 400
 Body: {"detail": null, "id": 1, "jsonrpc": "2.0", "result": 
       {"error": {"code": 10, "message": "ERR10: Could not verify signature!"},
        "status": false}, "time": 1622017905.9697392, "version": "privacyIDEA 3.6.dev2", "signature": <some signature>}
~~~~

And the following error is produced on the server side:
~~~~
[INFO][privacyidea.lib.tokens.pushtoken:696] The following error occurred during the signature check: 
"ConfigAdminError(description='Could not find gateway definition with identifier "poll only"', id=302)"
~~~~

I guess the error is caused by this line:
https://github.com/privacyidea/privacyidea/blob/2160e0f09db181f071d4cd02a11060f5f100ea59/privacyidea/lib/tokens/pushtoken.py#L675

In theory it might be possible to omitt this call for poll-only tokens, but here
https://github.com/privacyidea/privacyidea/blob/2160e0f09db181f071d4cd02a11060f5f100ea59/privacyidea/lib/tokens/pushtoken.py#L187
the configuration is needed to get the URL where push tokens answer to. (If I understand correctly).

**Comment by cornelinux** [comment]
@timosturm I think I fixed this in #2745. Can you please verify?

**Comment by timosturm** [comment]
@cornelinux It seems to work fine now.
